### DISCUSSION TITLE
fly -t or fly --target not found for set-pipeline

### DISCUSSION THREAD

**Discussion by DmitryKoichev** [concourse/concourse#7031] [General]
![concourse_fly_t](https://user-images.githubusercontent.com/11480235/118697822-62212f80-b818-11eb-9a63-7a598e0382eb.png)

can't understand how it can be? 
fly --target works fine for login but doesn't work for set-pipeline.
Could you please help fix this?

**Comment by taylorsilva** [comment]
Which version are you using? I tried this locally and it reads the `--target`  correctly. Is your `fly` aliased? `which fly`

**Reply by taylorsilva** [reply]
<img width="739" alt="image" src="https://user-images.githubusercontent.com/4583837/118728390-16638b80-b802-11eb-896d-e42b2425ccab.png">

**Reply by DmitryKoichev** [reply]
![Screenshot 2021-05-19 at 10 36 28](https://user-images.githubusercontent.com/11480235/118774031-13ae7800-b88e-11eb-99d8-d3361b5eee69.png)

**Reply by DmitryKoichev** [reply]
It worked perfectly previous weeks. 2 days ago I faced this problem

**Reply by taylorsilva** [reply]
That is odd. Sounds like something changed with your zsh profile though I'm not even sure what would have caused this behaviour. It's very odd. Sorry I don't have any other advice to provide :(
